connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -525,6 +525,7 @@ private WorkerTask buildWorkerTask(ClusterConfigState configState,
         producerProps.put(ProducerConfig.ACKS_CONFIG, "all");
         producerProps.put(ProducerConfig.MAX_IN_FLIGHT_REQUESTS_PER_CONNECTION, "1");
         producerProps.put(ProducerConfig.DELIVERY_TIMEOUT_MS_CONFIG, Integer.toString(Integer.MAX_VALUE));
+        producerProps.put(ProducerConfig.CLIENT_ID_CONFIG, "connect-producer-" + id);
```

Comment:
My only suggestion is to maybe consider `connector-` rather than `connect-`, since the producer (and consumer) are used for connectors rather than for the worker's own internal topic producers & consumers.
```suggestion
        producerProps.put(ProducerConfig.CLIENT_ID_CONFIG, "connector-producer-" + id);
```

Reply:
Thanks, that makes sense.